docs/user/metrics/github.csv

```diff
@@ -6,6 +6,7 @@ Library,Contributors,Forks,Watchers,Stars,Issues,PRs,Releases,Commits,Core SLOCs
 `samgeo`_,22,333,60,"3,232",157,143,39,236,"8,130",92,22%,MIT
 `TerraTorch`_,21,53,13,329,160,389,12,"1,380","22,577","4,191",60%,Apache-2.0
 `SITS`_,15,82,27,495,706,610,45,"6,596","26,194","9,474",94%,GPL-2.0
+`srai`_,14,21,11,271,82,12,36,314,"7,238","6,485",92%,Apache-2.0
```

Comment:
```suggestion
`srai`_,14,21,11,271,245,265,36,314,"7,238","6,336",92%,Apache-2.0
```
We count both open and closed issues/PRs. I also removed JSON SLOCs from the Test SLOCs total.